src/osd/PG.cc

```diff
@@ -4262,6 +4263,11 @@ void PG::scrub_process_inconsistent()
 	  }
 	}
       }
+
+      scrubber.authoritative.clear();
+      scrubber.inconsistent.clear();
+      scrubber.missing.clear();
```

Comment:
As I indicated above, we might as well move scrub_process_inconsistent unconditionally up into the per-chunk part.  In that case, these will be cleared here, and we can assert in scrub_finish that they are empty (we need to leave the clears in scrub_clear_state in case we exit scrub without finishing).